### PR TITLE
Automatic installation of Portable Git (Windows only)

### PR DESCRIPTION
**Pull request by laurentheirendt**
Functions for installing and updating automatically and without any user interaction `Portable Git` on any Windows system that is not properly configured.

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: initCobraToolbox.m [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function initCobraToolbox()
+function [] = initCobraToolbox()
```
Comment (syarra) [inline initCobraToolbox.m]: @laurentheirendt, could you remove the extra `[] = ` in the function signatures?

File: src/base/install/getGitBashVersion.m [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+%    installedVersion:      String of installed version of gitBash in the form `2.13.2`
+%    installedVersionNum:   Integer of `installedVersion`, e.g. `2132`
+%
+
+    global CBTDIR
+    global gitBashVersion
+
+    % define the architecture
+    archstr = computer('arch');
+    archBit = archstr(end-1:end);
```
Comment (syarra) [inline src/base/install/getGitBashVersion.m]: The variable `archBit` is not used in this script! Could you remove it?

File: src/base/install/installGitBash.m [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+%     installGitBash()
+%
+
+    global CBTDIR
+    global gitBashVersion
+
+    if ispc
+        % initialize variables
+        archstr = computer('arch');
+        archBit = archstr(end-1:end);
```
Comment (syarra) [inline src/base/install/installGitBash.m]: The variable `archBit` is not used in this script! Could you remove it?

File: src/base/install/portableGitSetup.m [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+                % session path
+                oldSessionPath = getenv('Path');
+                newSessionPath = strrep(oldSessionPath, [pathPortableGitFragments{i} ';'], '');
+                setenv('Path', newSessionPath);
+            end
+        end
+
+        % extract the archive and set the paths
+ 		if exist(fileNamePortableGit, 'file') == 2
```
Comment (syarra) [inline src/base/install/portableGitSetup.m]: Indentation is not consistent here!

File: initCobraToolbox.m (hunk 1 above)
Comment (laurentheirendt) [inline initCobraToolbox.m]: Done.

File: src/base/install/getGitBashVersion.m (hunk 2 above)
Comment (laurentheirendt) [inline src/base/install/getGitBashVersion.m]: Done.

File: src/base/install/installGitBash.m (hunk 3 above)
Comment (laurentheirendt) [inline src/base/install/installGitBash.m]: Done.

File: src/base/install/portableGitSetup.m (hunk 4 above)
Comment (laurentheirendt) [inline src/base/install/portableGitSetup.m]: Done.

**Pull request merged**